docs/topics/tour/kotlin-tour-control-flow.md

```diff
@@ -0,0 +1,239 @@
+</microformat>
+
+Like other programming languages, Kotlin is capable of checking conditional expressions and creating loops.
+
+## Conditional expressions
+
+Kotlin provides `if` and `when` for checking conditional expressions. If you have to choose between the two, we recommend
+using `when` as it leads to more robust and safer programs.
+
+### If
+
+To use `if`, add the conditional expression within parentheses and the action to take if the result is true within curly braces `{}`:
+
+```kotlin
+fun main() {
+//sampleStart
+    val d: Int
+    val check = true
+
+    if (check) {
+        d = 1
+    } else {
+        d = 2
+    }
+
+    println(d)
+    //1
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-if-kotlin"}
+
+There is no ternary operator `condition ? then : else` in Kotlin. Instead, `if` can be used as an expression. When using
+`if` as an expression, there are no curly braces `{}`:
+
+```kotlin
+fun main() { 
+//sampleStart
+    val a = 1
+    val b = 2
+
+    println(if (a > b) a else b) // Returns a value: 2
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-if-expression-kotlin"}
+
+### When
+
+Use `when` when you have a conditional expression with multiple branches.
+`when` can be used either as a statement or as an expression.
+
+Below is an example of using `when` as a statement:
+* Place the conditional expression within parentheses and the actions to take
+within curly braces `{}`. 
+* Use `->` in each branch to separate each condition from each action.
+
+```kotlin
+fun main() {
+//sampleStart
+    val obj = "Hello"
+
+    when (obj) {
+        "1" -> println("One")          //Checks whether obj equals to "One"
+        "Hello" -> println("Greeting") //Checks whether obj equals to "Hello"
+        else -> println("Unknown")     //Default statement
+    }
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-when-statement-kotlin"}
+
+Note that all branch conditions are checked sequentially until one of them is satisfied. So only the first suitable 
+branch is executed.
+
+Below is an example of using `when` as an expression. The `when` syntax is assigned immediately to a variable.
+
+```kotlin
+fun main() {
+//sampleStart    
+    val obj = "Hello"    
+    
+    val result = when (obj) {                                     
+        "1" -> "One"            //If obj equals "1", sets result to "one"
+        "Hello" -> "Greeting"   //If obj equals "Hello", sets result to "Greeting"
+        else -> "Unknown"       //Sets result to "Unknown" if no previous condition is satisfied
+    }
+    println(result)
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-when-expression-kotlin"}
+
+If when is used as an expression, the else branch is mandatory, unless the compiler can detect that all possible cases 
+are covered by the branch conditions.
+
```

Comment:
I like how the "when" expression works with Booleans, Enums, and Sealed classes. It may be a bit out of scope for this tutorial, WDYT?

Reply:
I added an example of how to use `when` with boolean expressions. As discussed over Slack, I'll add examples of how to use `when` with Enum and Sealed classes when those topics are covered in later versions of the tour 👍